### ISSUE TITLE
Add the optional argument for removing tildes

### ISSUE THREAD

**Issue by jumon** [ikegami-yukino/neologdn#21]
This issue is similar to the issue #10 . I assume we don't have the way we make tildes remain in a sentence when using `neologdn.normalize(sentence)` for now. I would like the optional argument like `remove_tildes` as the argument of `neologdn.normalize`. Below is an example of what my suggestion might look like.
```
>>> neologdn.normalize("応仁の乱は1467〜1487年の11年間に渡って続きました。")
'応仁の乱は14671487年の11年間に渡って続きました。'
>>> neologdn.normalize("応仁の乱は1467〜1487年の11年間に渡って続きました。", remove_tildes=False)
'応仁の乱は1467〜1487年の11年間に渡って続きました。'
```
Like this example, I would sometimes prefer tildes to remain in a sentence. If there is no problems in terms of speed or something like that, I think this option is useful.

**Comment by yokoshin** [comment]
私も同じ問題があり、少し直してみようかと思っています。
こんなインターフェイスでいかがですか。
```python
        # そのまま
        self.assertEqual(normalize('チルダ~∼∾〜〰～', tilda="ignore"), 'チルダ~∼∾〜〰～')
        # ノーマライズ(半角)
        self.assertEqual(normalize('チルダ~∼∾〜〰～', tilda="normalize", repeat=0), 'チルダ~~~~~~')
        # ノーマライズ(全角)
        self.assertEqual(normalize('チルダ~∼∾〜〰～', tilda="normalize_zen", repeat=0), 'チルダ～～～～～～')
        # 削除
        self.assertEqual(normalize('チルダ~∼∾〜〰～', tilda="remove"), 'チルダ')
        #デフォルトはremove
        self.assertEqual(normalize('チルダ~∼∾〜〰～'), 'チルダ')
```

**Comment by ikegami-yukino** [comment]
@jumon @yokoshin 
I resolved this problem at version 0.5.
Thanks a lot!
